app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -0,0 +1,146 @@
+package protect.card_locker
+
+import android.os.Bundle
+import android.text.Spanned
+import android.view.MenuItem
+import android.view.View
+import android.widget.ScrollView
+import android.widget.TextView
+
+import androidx.annotation.StringRes
+
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+import protect.card_locker.databinding.AboutActivityBinding
+
+class AboutActivity : CatimaAppCompatActivity() {
+    private companion object {
+        private const val TAG = "Catima"
+    }
+
+    private var _binding: AboutActivityBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var content: AboutContent
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        _binding = AboutActivityBinding.inflate(layoutInflater)
+        content = AboutContent(this)
+        title = content.pageTitle
+        setContentView(binding.root)
+        setSupportActionBar(binding.toolbar)
+        enableToolbarBackButton()
+
+        binding.apply {
+            creditsSub.text = content.copyrightShort
+            versionHistorySub.text = content.versionHistory
+
+            versionHistory.tag = "https://catima.app/changelog/"
+            translate.tag = "https://hosted.weblate.org/engage/catima/"
+            license.tag = "https://github.com/CatimaLoyalty/Android/blob/main/LICENSE"
+            repo.tag = "https://github.com/CatimaLoyalty/Android/"
+            privacy.tag = "https://catima.app/privacy-policy/"
+            reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
+            rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"
+            donate.tag = "https://catima.app/donate"
+
+            // Hide Google Play rate button if not on Google Play
+            rate.visibility = if (BuildConfig.showRateOnGooglePlay) View.VISIBLE else View.GONE
+            // Hide donate button on Google Play (Google Play doesn't allow donation links)
+            donate.visibility = if (BuildConfig.showDonate) View.VISIBLE else View.GONE
+        }
+
+        bindClickListeners()
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (item.itemId == android.R.id.home) {
+            finish()
+        }
+        return super.onOptionsItemSelected(item)
```

Comment:
You can consider using a `when` block here for a cleaner and easily extensible code.

```suggestion
        return when (item.itemId) {
            android.R.id.home -> finish()
            else -> super.onOptionsItemSelected(item)
        }
```

Reply:
updated with @theimpulson suggestion